### ISSUE TITLE
The build of the project is broken in Travis.

### ISSUE THREAD

**Issue by aivinog1** [yegor256/cactoos#1172]
You can see this PR: https://github.com/yegor256/cactoos/pull/1157

The build of the project in Travis doesn't work. See this build: https://travis-ci.org/yegor256/cactoos/builds/568301019?utm_source=github_status&utm_medium=notification

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by 0crat** [comment]
@aivinog1[/z](https://www.0crat.com/u/aivinog1) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C63314D6Z) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by vzurauskas** [comment]
@llorllale I have this problem in my PR as well: https://github.com/yegor256/cactoos/pull/1171

**Comment by vzurauskas** [comment]
@llorllale This is blocking the development.

**Comment by vzurauskas** [comment]
@llorllale Another PR blocked by this: https://github.com/llorllale/cactoos-matchers/pull/140

**Comment by Glusk2** [comment]
The error message int the Travis [build log](https://travis-ci.org/yegor256/cactoos/builds/578062519) reads:
>`Expected feature release number in range of 9 to 14, but got: 8`

This can be fixed in two ways [[1]](https://travis-ci.community/t/error-installing-oraclejdk8-expected-feature-release-number-in-range-of-9-to-14-but-got-8/3766/8):

- Add `dist: trusty` to [`.travis.yml`](https://github.com/yegor256/cactoos/blob/master/.travis.yml)
- Migrate to OpenJDK builds

**Comment by paulodamaso** [comment]
@0crat assign @iakunin

**Comment by 0crat** [comment]
> @0crat assign @iakunin ([here](https://github.com/yegor256/cactoos/issues/1172#issuecomment-530501457))

@paulodamaso The job [#1172](https://github.com/yegor256/cactoos/issues/1172) assigned to @iakunin[/z](https://www.0crat.com/u/iakunin), here is [why](https://www.0crat.com/footprint/C63314D6Z/a5d45b4c-8634-40c1-915c-8d2d172cbf1b); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @aivinog1[/z](https://www.0crat.com/u/aivinog1)

**Comment by paulodamaso** [comment]
@aivinog1 please take a look

**Comment by aivinog1** [comment]
@paulodamaso as we can see [there](https://travis-ci.org/yegor256/cactoos/builds/585379348?utm_source=github_status&utm_medium=notification) for now build is passed.

I'm closing this.

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) all `QA`s are on vacation. Please handle that.

**Comment by 0crat** [comment]
The job [#1172](https://github.com/yegor256/cactoos/issues/1172) is now out of scope

**Comment by iakunin** [comment]
@paulodamaso, is it ok that I didn’t receive any reputation for this issue?

**Comment by paulodamaso** [comment]
@0crat quality good

**Comment by iakunin** [comment]
@paulodamaso seems like it didn’t help

**Comment by paulodamaso** [comment]
@iakunin Please report it to https://github.com/zerocracy/farm repositoy

**Comment by iakunin** [comment]
@paulodamaso I've done it: https://github.com/zerocracy/farm/issues/2157

**Comment by paulodamaso** [comment]
@iakunin Great, let's see what they do about this

**Comment by iakunin** [comment]
@paulodamaso they fixed it: https://github.com/zerocracy/farm/issues/2155

Let's try one more time to obtain my reputation points :)

**Comment by paulodamaso** [comment]
@0crat quality good

**Comment by 0crat** [comment]
> @0crat quality good ([here](https://github.com/yegor256/cactoos/issues/1172#issuecomment-534141092))

@paulodamaso There is no quality review for [#1172](https://github.com/yegor256/cactoos/issues/1172), no performer
